README.md

```diff
@@ -38,4 +38,11 @@ Green Light Health App helps businesses prevent spread of illness in their locat
 # Product Backlog:
 Find the [Product Backlog](https://github.com/jorgecotillo/green_light_health/projects/1)
 under github projects.
+
 # Definition of Ready:
+Our D.O.R. requires that the team be able to determine "what needs to be done" and the "amount of work" required to complete a given story.
+The below checklists should be complete to ensure a story is Ready.
+- [ ] Does the story capture and describe a reasonable and unit of work?
```

Comment:
did you mean: "...a reasonable unit of work?" Based on what professor mentioned, everytime we see the word "and" it means a separate item, should we split this item into two?

Reply:
Yup it's a typo, will address.